ee/codegen/src/__test__/nodes/code-execution-node.test.ts

```diff
@@ -367,4 +372,40 @@ describe("CodeExecutionNode", () => {
       expect(await writer.toStringFormatted()).toMatchSnapshot();
     });
   });
+
+  describe.skip("basic with ports", () => {
+    beforeEach(async () => {
+      const ifPortId = "3e9a8943-db9d-4586-a657-3f4ce1c1b595";
+      const elsePortId = "27aee210-2375-43e0-8dad-8856d9ff3cfb";
+      const nodePortsData: NodePort[] = [
+        nodePortFactory({
+          type: "IF",
+          id: ifPortId,
+        }),
+        nodePortFactory({
+          type: "ELSE",
+          id: elsePortId,
+        }),
+      ];
+
+      const nodeData = codeExecutionNodeFactory()
+        .withPorts(nodePortsData)
```

Comment:
[q] Why this over `codeExecutionNodeFactory({ports: nodePortsData})`? just curious, cool with new pattern too